### PR TITLE
Enabled default template to be replaced

### PR DESCRIPTION
**Pull request by samueldjack**
This pull request enables the default template of the custom element to be replaced, making it possible to use the wrap feature (see "Custom Elements and Input Groups" [here](https://chmln.github.io/flatpickr/)).

### CODE REVIEW

File: src/aurelia-flatpickr.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export class AureliaFlatpickrCustomElement {
         for(let d = 0; d < modelDates.length; d++) {
             let modelDate = modelDates[d];
 
-            if (view.indexOf(modelDate) > -1) {
```
Comment (istrau2) [inline src/aurelia-flatpickr.js:80]: Why the change from indexOf to some check?

**Comment by istrau2** [comment]
Thanks for the PR, it is appreciated. Any way you could add a quick example into the Readme on how to use the custom element with the wrap feature?

File: src/aurelia-flatpickr.js (hunk 1 above)
Comment (samueldjack) [reply src/aurelia-flatpickr.js:80]: In my testing, indexOf wasn't recognising two identical as being equal, which is why I switched to using getTime()

**Comment by samueldjack** [comment]
Certainly!

File: src/aurelia-flatpickr.js (hunk 1 above)
Comment (istrau2) [reply src/aurelia-flatpickr.js:80]: @samueldjack I checked this and you are right, the method is buggy. I have added a commit that fixes the bugs in this method and adds testing in general and specifically, testing for this method. I don't really want to use .some() because there is no need to search for more than one match in that instance. Instead I am using findIndex(). Let me know you think.
